src/constants/fonts.scss

```diff
@@ -0,0 +1,75 @@
+@import './colors.scss';
+
```

Comment:
change to groups of styles.

ex:

$desktop-h1 {
  font-family: $heading-font-family;
  font-weight: 200;
  font-size: 32px;
  color: $primary-black;
}